provisioning/provision.sh

```diff
@@ -53,6 +53,7 @@ virtualenv /dissemin/.vm_venv --no-site-packages -p $(which python2.7)
 # We install dependencies in the virtualenv
 req_files=(requirements.txt requirements-dev.txt)
 
+/dissemin/.wm_venv/bin/pip install --upgrade setuptools
```

Comment:
It worked because you might not need to update setup tools :p

On Sat, Jul 23, 2016, 10:37 Kenji Maillard <notifications@github.com> wrote:

> In provisioning/provision.sh
> <https://github.com/dissemin/dissemin/pull/302#discussion_r71969549>:
>
> > @@ -53,6 +53,7 @@ virtualenv /dissemin/.vm_venv --no-site-packages -p $(which python2.7)
> >  # We install dependencies in the virtualenv
> >  req_files=(requirements.txt requirements-dev.txt)
> >
> > +/dissemin/.wm_venv/bin/pip install --upgrade setuptools
>
> Sorry for the mistake.
> ...But now I don't understand why it worked on my computer ^^'
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/dissemin/dissemin/pull/302/files/840645a7f474c2e3052c1a5543eb9025a2295761#r71969549>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AATMxFHuwUxhUsown3ssptHtBH62J3p7ks5qYdK5gaJpZM4JTRJZ>
> .
>
-- 

-- 

Ryan Lahfa, from my phone.